src/scenes/Root/Root.tsx

```diff
@@ -30,6 +31,7 @@ export const Root = () => {
       <Route path="/" element={<Home />} />
       <Route path="/organizations/*" element={<Organizations />} />
       <Route path="/projects/*" element={<Projects />} />
+      <Route path="/Languages/*" element={<Languages />} />
```

Comment:
URLs are always lowercase.
```suggestion
      <Route path="/languages/*" element={<Languages />} />
```